From a5e591d52c879af97d29ddf0f1f1ba23a15a0c6e Mon Sep 17 00:00:00 2001
From: Girish Kumar M P <quic_gmp@quicinc.com>
Date: Mon, 1 Apr 2024 16:43:57 +0530
Subject: [PATCH] wifi: ath12k: Increase dp tx ring buffer size to improve the
 performance

Observed low throughput in Multi-client Traffic
with existing dp tx ring buffer size configuration.

Increased the DP_TX_COMP_RING_SIZE and ATH12K_NUM_POOL_TX_DESC,
to improve throughput in multi-client traffic.

Signed-off-by: Girish Kumar M P <quic_gmp@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/dp.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/ath/ath12k/dp.h b/drivers/net/wireless/ath/ath12k/dp.h
index e859f8a..0bfba16 100644
--- a/drivers/net/wireless/ath/ath12k/dp.h
+++ b/drivers/net/wireless/ath/ath12k/dp.h
@@ -158,11 +158,11 @@ struct ath12k_pdev_dp {
 
 #ifdef CPTCFG_ATH12K_MEM_PROFILE_512M
 
-#define DP_TX_COMP_RING_SIZE		8192
+#define DP_TX_COMP_RING_SIZE		16384
 #define DP_RXDMA_MON_STATUS_RING_SIZE	512
 #define DP_RXDMA_MONITOR_BUF_RING_SIZE	256
 #define DP_RXDMA_MONITOR_DST_RING_SIZE	512
-#define ATH12K_NUM_POOL_TX_DESC		8192
+#define ATH12K_NUM_POOL_TX_DESC		16384
 #define DP_REO2PPE_RING_SIZE		2048
 /* TODO revisit this count during testing */
 #define ATH12K_RX_DESC_COUNT		(4096)
-- 
2.34.1

